src/interfaces.cairo

```diff
@@ -38,3 +40,15 @@ pub trait IMarket<TContractState> {
     fn repay(ref self: TContractState, token: ContractAddress, amount: felt252);
     fn repay_all(ref self: TContractState, token: ContractAddress);
 }
+
+#[starknet::interface]
+pub trait IAirdrop<TContractState> {
```

Comment:
Remove all unused methods from interface as they go in the line count for audit. Always try to keep as little code as possible.